mzLib/FlashLFQ/PeakIndexing/Interfaces/IFlashLfqIndexingEngine.cs

```diff
@@ -0,0 +1,24 @@
+﻿using MassSpectrometry;
+using MzLibUtil;
+using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+
+namespace FlashLFQ.Interfaces
+{
+    /// <summary>
+    /// FlashLFQ requires indexing engines to IndexPeaks, serialize and deserialize indexed peaks,
+    /// and retrieve specific peaks (m/z, index pair) or xics (m/z at and around a given retention time
+    /// </summary>
+    public interface IFlashLfqIndexingEngine
```

Comment:
Comments on methods would be lovely. Summary comments on interface methods will pop up in Visual Studio when mousing over the method called on a concrete type. This ensure you do not need to make comments on every implementing IFlashLfqIndexingEngine